<?xml version="1.0" ?>
<launch>

    <param name="leg_dist_model" value="$(find player_tracker)/model/leg_distance-gmm.pkl"/>

    <!-- run particle_filter launch-prefix="xterm -e gdb -args" -->
    <node pkg="player_tracker" type="particle_filter" name="particle_filter" output="screen" respawn="true"/>
    <node pkg="player_tracker" type="evidence_extractor.py" name="evidence_extractor" output="screen"/> 
    <node pkg="player_tracker" type="blob_publisher" name="blob_publisher" output="screen"  required="true" />

    <node pkg="player_tracker" type="test_local_occupancy_grid_mapping" name="test_local_occupancy_grid_mapping" output="screen"/>

    <!-- run rviz -->
    <node pkg="rviz" type="rviz" name="rviz" />
    <!--  /home/ewerlopes/Desktop/_2018-08-30-16-10-13.bag"/ -->
    <!-- play rosbag normal rate param is 1-->

    <!-- bags: 
        * VERY_HARD: _2018-08-29-11-15-45.bag
        * EASY: _2018-08-30-16-10-13.bag    
    -->
    <node pkg="rosbag" type="play" name="player" output="screen" required="true" 
        args="--delay=4
        --clock 
        --rate=1
        -s 0
        /home/airlab/Scrivania/_2018-08-30-16-10-13.bag" />
    <param name="use_sim_time" value="true" />

</launch>
